src/parsers/manifest/types.ts

```diff
@@ -46,11 +46,12 @@ export interface IParsedAdaptation {
   type: string;
 
   // optional
+  trickModeTrack? : IParsedAdaptation;
+  isTrickModeFor? : string;
```

Comment:
Wait.. Why would both be needed?
I'm OK with one or the other, but I don't see the point here.